### ISSUE TITLE
Projected coordinates

### ISSUE THREAD

**Issue by ChrisBarker-NOAA** [pyugrid/pyugrid#52]
It would be nice for py_ugrid to support projected coordinates. For example we've gotten some SUNTANS output that is using projected easting-northing, rather than lat-lon.

It seems this is supported in CF,a nd should be in UGRID eventually:
https://github.com/ugrid-conventions/ugrid-conventions/issues/12

At this point, I think lat-lon is only assumed in the reading / writing code, so should be huge impacts.

But I'm not entirely sure at what level to support it:
 - would a UGrid object have the projection info?
 - would we put in any hooks to re-project, etc? or leave that to external code.

How does Iris handle this?

**Comment by ocefpaf** [comment]
@ChrisBarker-NOAA I would defer that to a CF-interpreter or users input.  As long as the `UGRID` conventions say that projected coordinates are OK and `pyugrid` can find them, it is up to the user to explore the metadata and figure out what to do.

`xarray` does something very simple when it comes to metadata like that.  It just reads them and put in a big dictionary where it is the user responsibility to check it and decide what to do.  The whole reason to do that is to avoid implementing a CF parser.

`iris` (I belive) can interprete that making this even easier.

TL;DR

I would keep `pyugrid` simple and avoid implementing that.

**Comment by ChrisBarker-NOAA** [comment]
OK, it may "just work" as it is anyway -- but I'd rather than the py_ugird API was complete, the user shouldn't have to go examine the netcdf file directly to know what things mean -- and there may not BE a netcdf file anyway.

But let's cross that bridge when we come to it.

-CHB
